rs/execution_environment/src/scheduler/test_utilities.rs

```diff
@@ -197,9 +197,12 @@ impl SchedulerTest {
     }
 
     pub fn execution_cost(&self, num_instructions: NumInstructions) -> Cycles {
-        self.scheduler
-            .cycles_account_manager
-            .execution_cost(num_instructions, self.subnet_size())
+        const IS_WASM64_EXECUTION: bool = false;
```

Comment:
Nit: Is this const really necessary? We use it directly below and it's a test function.

Reply:
fixed.